Analytics/Integrations/Kahuna/SEGKahunaIntegration.h

```diff
@@ -14,15 +14,17 @@
 @property (nonatomic, assign) BOOL valid;
 @property (nonatomic, assign) BOOL initialized;
 @property (nonatomic, copy) NSDictionary *settings;
+@property Class kahunaClass;
 
 @end
 
 
 @interface SEGKahunaPushMonitor : NSObject
-@property (atomic) NSDictionary *pushInfo;
-@property (atomic) UIApplicationState applicationState;
-@property (atomic) BOOL kahunaInitialized;
+@property (nonatomic) NSDictionary *pushInfo;
+@property (nonatomic) UIApplicationState applicationState;
+@property (nonatomic) BOOL kahunaInitialized;
```

Comment:
I vaguely remembered some of these properties were changes from nonatomic to atomic.  Why are we changing them back?